app/widgets/common/deployModal/EnvironmentDropdown/EnvironmentDropdown.types.ts

```diff
@@ -0,0 +1,19 @@
+import type { Deployment } from '../../deploymentsView/types';
+
+export interface FetchedEnvironment {
+    id: string;
+    // eslint-disable-next-line
+    display_name: string;
+    capabilities: Deployment['capabilities'];
+}
+
+export interface Environment {
+    id: string;
+    capabilities: Deployment['capabilities'];
+    displayName: string;
+}
```

Comment:
WDYT about:
```suggestion
export interface FetchedEnvironment {
    id: Deployment['id'];
    // eslint-disable-next-line
    display_name: Deployment['display_name'];
    capabilities: Deployment['capabilities'];
}

export interface Environment {
    id: FetchedEnvironment['id'];
    capabilities: FetchedEnvironment['capabilities'];
    displayName: FetchedEnvironment['display_name'];
}
```
?

That would a bit clarify the relationships IMHO.

Reply:
Seems like a great idea to me :)
Applying it right away 🎸